src/NuGet.Indexing/PackageMetadataExtraction.cs

```diff
@@ -93,6 +94,9 @@ public static void MakePackage(Stream stream, IDictionary<string, string> packag
             ExtractProperty(package, document, "licenseUrl");
             ExtractProperty(package, document, "requiresLicenseAcceptance");
             ExtractDependencies(package, document);
+
+            package["published"] = DateTime.UtcNow.ToString("O");
```

Comment:
can we switch to using DateTimeOffset instead of DateTime. Generally DateTimeOffset is recommended to be used whenever you can instead of DateTime.